src/main/java/com/tutorbridge/TutorBridge/Users/UserTable.java

```diff
@@ -0,0 +1,47 @@
+package com.tutorbridge.TutorBridge.Users;
+
+import jakarta.persistence.*;
+//import jakarta.persistence.;
+
+@Entity
+@Table
+public class UserTable {
+    @Id
+    private String hash_val;
```

Comment:
hash_val should just be called "ID" This refers to the unique table ID of the user, and it's common practice to always call it ID. ID also **always** has to be an integer.